charts/k8s-service/README.md

```diff
@@ -981,6 +981,30 @@ canary:
 ```
 Once deployed, your service will route traffic across both your stable and canary deployments, allowing you to monitor for and catch any issues early.
 
+## How do I verify my canary deployment? 
```

Comment:
I need to verify these commands / output once I have my code changes implemented and running locally / in my GKE staging environment - but adding them here for now.

Reply:
Confirmed locally - these docs are correct and the rollback procedure works.